images/step-issuer/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
+# step-issuer
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/step-issuer` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/step-issuer/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+[step-issuer](https://smallstep.com/docs/step-issuer) is a certificate issuer for cert-manager using step certificates CA
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/step-issuer:latest
+```
+<!--getting:end-->
+
+<!--body:start--><!--body:end-->
```

Comment:
Can we add some usage docs here in the body section? i.e how to use the image with helm/docker etc, and to override with our image?

Reply:
Added a brief description on where they can find the proper resources to set up their step-issuer image.